src/Protocol/MojangAPI.cpp

```diff
@@ -762,45 +566,16 @@ void cMojangAPI::QueryUUIDToProfile(const cUUID & a_UUID)
 	AString Address = m_UUIDToProfileAddress;
 	ReplaceURL(Address, "%UUID%", a_UUID.ToShortString());
 
-	// Create the HTTP request:
-	AString Request;
-	Request += "GET " + Address + " HTTP/1.0\r\n";  // We need to use HTTP 1.0 because we don't handle Chunked transfer encoding
-	Request += "Host: " + m_UUIDToProfileServer + "\r\n";
-	Request += "User-Agent: Cuberite\r\n";
-	Request += "Connection: close\r\n";
-	Request += "Content-Length: 0\r\n";
-	Request += "\r\n";
-
-	// Get the response from the server:
-	AString Response;
-	if (!SecureRequest(m_UUIDToProfileServer, Request, Response))
-	{
-		return;
-	}
-
-	// Check the HTTP status line:
-	const AString Prefix("HTTP/1.1 200 OK");
-	AString HexDump;
-	if (Response.compare(0, Prefix.size(), Prefix))
-	{
-		LOGINFO("%s failed: bad HTTP status line received", __FUNCTION__);
-		LOGD("Response: \n%s", CreateHexDump(HexDump, Response.data(), Response.size(), 16).c_str());
-		return;
-	}
-
-	// Erase the HTTP headers from the response:
-	size_t idxHeadersEnd = Response.find("\r\n\r\n");
-	if (idxHeadersEnd == AString::npos)
+	// Create and send the HTTP request
+	auto [Success, Response] = cUrlClient::BlockingGet(Address);
+	if (!Success)
 	{
-		LOGINFO("%s failed: bad HTTP response header received", __FUNCTION__);
-		LOGD("Response: \n%s", CreateHexDump(HexDump, Response.data(), Response.size(), 16).c_str());
 		return;
 	}
-	Response.erase(0, idxHeadersEnd + 4);
 
 	// Parse the returned string into Json:
 	Json::Value root;
-	AString ParseError;
+	AString ParseError, HexDump;
```

Comment:
Seems like this is not used anymore, debugging leftovers?

Reply:
Right, I saw no changes below this line, so I thought, why was it here. Didn't notice the variable used to be declared above, which you removed. Sorry about that.